src/osd/OSD.h

```diff
@@ -636,18 +636,20 @@ class OSDService {
   /// Note: required, as in Crimson this operation is 'futurized'.
   void queue_scrub_maps_compared(PG* pg, Scrub::scrub_prio_t with_priority);
 
-  void queue_for_rep_scrub(PG* pg,
-			   Scrub::scrub_prio_t with_high_priority,
-			   unsigned int qu_priority);
+  void queue_for_rep_scrub(PG* pg, Scrub::scrub_prio_t with_high_priority,
+                           unsigned int qu_priority,
+                           Scrub::act_token_t act_token);
 
   /// Signals a change in the number of in-flight recovery writes
-  void queue_scrub_replica_pushes(PG *pg, Scrub::scrub_prio_t with_priority);
+  void queue_scrub_replica_pushes(PG* pg, Scrub::scrub_prio_t with_priority);
```

Comment:
Stray change, and PG *pg is correct.

Reply:
Stray indeed (but improvement, to my taste. The *type* of 'pg' is a 'pointer to PG".). WIll reset.

Reply:
Style guide apparently says that either is permitted.  Still, best to avoid whitespace changes mixed with a larger commit.